src/admin-views/tribe-options-tickets.php

```diff
@@ -176,27 +157,29 @@
 
 if ( tribe_get_option( 'ticket-paypal-enable', true ) ) {
 	$pages = get_pages( array( 'post_status' => 'publish', 'posts_per_page' => - 1 ) );
+
 	if ( ! empty( $pages ) ) {
 		$pages        = array_combine( wp_list_pluck( $pages, 'ID' ), wp_list_pluck( $pages, 'post_title' ) );
 		$default_page = reset( $pages );
 	} else {
 		$pages        = array( 0 => __( 'There are no published pages', 'event-tickets' ) );
 		$default_page = null;
 	}
+
 	$tpp_success_shortcode = 'tribe-tpp-success';
+
 	/**
-	 * Filters the available currency code options for PayPal
+	 * Filters the available Tribe__Tickets__Commerce__Currency currency code options for TPP
 	 *
 	 * @since TBD
-	 *
-	 * @param array $currency_code_options
 	 */
-	$currency_code_options = apply_filters( 'tribe_tickets_paypal_currency_code_options', $currency_code_options );
+	$paypal_currency_code_options = apply_filters( 'tribe_tickets_paypal_currency_code_options', Tribe__Tickets__Commerce__Currency::get_instance()->generate_currency_code_options() );
```

Comment:
use the container here, not the `get_instance` method

Reply:
even better, use `tribe_callback( 'tickets.commerce.currency', 'generate_currency_code_options' )` to hook a (lazy) lambda